src/interfaces/widget.ts

```diff
@@ -69,7 +69,8 @@ export interface WidgetOptions {
      */
     hierarchyDisableDescendants?: boolean
     hierarchyDisableParent?: boolean
-    actionGroups?: WidgetOperations
+    hierarchyShowRowActions?: boolean
```

Comment:
Can't this be handled with widget operations [`include` section](https://github.com/tesler-platform/tesler-schema/blob/572a65079df1f30e748a1d824e54a2fa46d2d839/src/interfaces/widget.ts#L124)? 

Provide an empty array when no actions should be available